.pre-commit-config.yaml

```diff
@@ -51,18 +51,18 @@ repos:
       - id: debug-statements
```

Comment:
Can you remove these 3 last debug-statement hooks?

Reply:
Thanks, there was a fourth one above that I also removed (see e9cb22dba1d9eec015e63458e698b4e5d5c10f0e)